Sources/OpenAPIServiceModel/ParseSchemas.swift

```diff
@@ -0,0 +1,202 @@
+                
+                model.structureDescriptions[enclosingEntityName] = structureDescription
+            }
+        case .array(_, let arrayContext):
+            parseArrayDefinitionSchemas(arrayMetadata: arrayContext, enclosingEntityName: &enclosingEntityName,
+                                        model: &model, modelOverride: modelOverride, document: document)
+        case .string(_, let stringContext):
+            addStringField(metadata: stringContext, schema: schema,
+                           model: &model, fieldName: enclosingEntityName, modelOverride: modelOverride)
+        case .number(_, let numberContext):
+            model.fieldDescriptions[enclosingEntityName] = Fields.double(rangeConstraint:
+                                                                            NumericRangeConstraint<Double>(minimum: numberContext.minimum?.value,
+                                                                                                           maximum: numberContext.maximum?.value,
+                                                                                                           exclusiveMinimum: numberContext.minimum?.exclusive ?? false,
+                                                                                                           exclusiveMaximum: numberContext.maximum?.exclusive ?? false))
+        case .all(let otherSchema, _), .any(let otherSchema, _), .one(let otherSchema, _):
+            var structureDescription = StructureDescription()
+            parseOtherSchemas(structureDescription: &structureDescription, enclosingEntityName: &enclosingEntityName, model: &model, otherSchema: otherSchema, modelOverride: modelOverride, document: document)
+        case .reference:
+            break
+        case .fragment:
+            fatalError("Schema 'fragment' not implemented")
+        case .not:
+            fatalError("Schema 'not' not implemented")
+        }
+    }
+    
+    static func parseObjectSchema(structureDescription: inout StructureDescription, enclosingEntityName: inout String,
+                                  model: inout OpenAPIServiceModel, objectContext: JSONSchema.ObjectContext,
+                                  modelOverride: ModelOverride?, document: OpenAPI.Document) {
+        let sortedKeys = objectContext.properties.keys.sorted(by: <)
+        
+        for (index, name) in sortedKeys.enumerated() {
+            guard let property = objectContext.properties[name] else {
+                continue
+            }
+            switch property {
+            case .reference(let ref):
+                if let referenceName = ref.name {
+                    structureDescription.members[name] = Member(value: referenceName, position: index,
+                                                                required: objectContext.requiredArray.contains(name),
```

Comment:
Can you comment here or add a `TODO: ...` specifying that once OpenAPIKit fixes the bug, the code here will be modified?

Reply:
Done.